tests/test_units/test_utils.py

```diff
@@ -0,0 +1,132 @@
+"""Test the functions of the utils.py module"""
+
+import pytest
+
+from fmu.dataio import _utils as utils
+
+
+def test_detect_inside_rms():
+
+    # TODO
+
+    pass
+
+
+def test_drop_nones():
+
+    # TODO
+
+    pass
```

Comment:
?

Reply:
When establishing the new test module `test_utils.py` I made placeholders for other tests, for others to fill in. Could be removed from this PR, of course.